docker-compose.e2e.yml

```diff
@@ -0,0 +1,69 @@
+services:
```

Comment:
Organization-wise, I really think this should live under `tests/e2e`. Is it easy to just re-configure all of the paths from there, or do any problems arise?

Reply:
Makes sense.